src/vs/languages/css/common/parser/cssParser.ts

```diff
@@ -889,7 +909,8 @@ export class Parser {
 			node.setExpression(this._parseStringLiteral()) ||
 			node.setExpression(this._parseNumeric()) ||
 			node.setExpression(this._parseHexColor()) ||
-			node.setExpression(this._parseOperation())
+			node.setExpression(this._parseOperation()) ||
+			node.setExpression(this._parseVariableExpression())
```

Comment:
This needs to be in fron of parseIndent. Actually i'd rather do not have a _parseVariableExpression but let variable declarations be handled by _parseFunction.